MD/md-20/README.md

```diff
@@ -0,0 +1,93 @@
+
+To summarize:
+- **L1 -> L2**: Lock->Mint->Swap
+- **L2 -> L1**: Swap->Lock->Mint
+
+It is worth noting that the term "Mint" on L1 is slightly inaccurate. The L1 ERC20 MOVE is already minted before any transfers occur, so no L1 mint transaction takes place during a bridge transfer. However, a mint _does_ occur on L2. The L2 ERC20 MOVE is either _locked in the pool_ or _withdrawn from the pool_. We can continue using this language as it is common across the organization.
+
+## Definitions
+
+- **Bridge Relayer**: Infrastructure used to maintain the synchronization of bridge request states.
+- **Initiator Contract**: Contract a user uses to initiate a bridge transfer.
+- **Counterparty Contract**: Contract on the opposite chain used to complete a bridge transfer.
+- **L2 $MOVE**: For clarity, the MOVE on L2. This token is equivalent in functionality and ability to the $APT token. It is the native token of L2 and is used to pay for transactions.
+- **BRIDGE-FA**: The asset that is minted and deposited into the user's account before the swap occurs. It is an Aptos [Fungible Asset](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/fungible_asset.move).
+- **L1 ERC20 $MOVE**: The ERC20 Move token on Ethereum (L1).
+- **Pool**: A holding place for assets or state data in the contract on either L1 or L2.
+
+## Desiderata
+
+### 1. Seamless Acquisition of L2 $MOVE on Completion of Transfer
+
+- **User Journey**: After a transfer is initiated and the secrets are revealed, the Bridge Relayer calls `complete_bridge_transfer`. At this point, the bridge move module mints the `BRIDGE-FA`, a temporary holding asset used to swap for L2 $MOVE.
+- **Justification**: Using the BRIDGE-FA provides a clean API and audited safety features. Furthermore, it has already been implemented. The prior implementation was called `MOVETH`, but the source code is the same. [MOVETH.move source code](https://github.com/movementlabsxyz/movement/blob/main/protocol-units/bridge/move-modules/sources/MOVETH.move). The [atomic_bridge_counterparty.move](https://github.com/movementlabsxyz/movement/blob/main/protocol-units/bridge/move-modules/sources/atomic_bridge_counterparty.move) module makes calls to the FA module.
+
+### 2. Safe and Secure Access to L2 $MOVE Pool
+
+- **Justification**: The Bridge Relayer keys should not have access to the $MOVE pool. If they did, they could arbitrarily mint L2 $MOVE, effectively offsetting the balance between L1 ERC20 $MOVE and L2 $MOVE, potentially incurring huge losses for Movement Labs.
+
+### 3. Gas Sponsorship for the L2 Swap
+
+- **User Journey**: After a transfer from L1 to L2 is completed, the user expects to receive L2 $MOVE equivalent to the ERC20 $MOVE that was transferred into their Movement Wallet.
+- **Justification**: At the initial step of the swap (when the transfer is completed on Movement L2), the user requires some L2 $MOVE to execute the swap transaction.
```

Comment:
Isn't the bridge relayer completing it for the user? The user would have the option to complete.

Reply:
Yes correct the bridge relayer completes it for the user.